src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -733,13 +732,8 @@ function ExpandedPostDetails({
   const t = useTheme()
   const pal = usePalette('default')
   const {_, i18n} = useLingui()
-  const openLink = useOpenLink()
   const isRootPost = !('reply' in post.record)
 
-  const onTranslatePress = React.useCallback(() => {
-    openLink(translatorUrl, true)
```

Comment:
This was here intentionally so that we can skip the consent modal for the translate flow